package.json

```diff
@@ -33,6 +33,7 @@
     "class-validator": "^0.14.1",
     "fastest-levenshtein": "^1.0.16",
     "ioredis": "^5.4.1",
+    "nest": "^0.1.6",
```

Comment:
why?

Reply:
No reason mate that file shouldnt be included in the commit.
My bad

Reply:
78e2de8a456475b200297f847e9ca29e581b6162